Calibration/HcalCalibAlgos/src/RecAnalyzerMinbias.cc

```diff
@@ -245,14 +326,35 @@ void RecAnalyzerMinbias::analyze(const edm::Event& iEvent, const edm::EventSetup
   const HFRecHitCollection HithfMB = *(hfMB.product());
   edm::LogInfo("AnalyzerMB") << "HF MB size of collection " << HithfMB.size();
   if (HithfMB.size() < 1700 && runNZS_) {
-    edm::LogWarning("AnalyzerMB") << "HF problem " << rnnum << " size "
+    edm::LogWarning("AnalyzerMB") << "HF problem " << rnnum_ << " size "
 				  << HithfMB.size();
     return;
   }
 
-  if (ignoreL1) {
-    analyzeHcal(HithbheMB, HithfMB, 1);
-  } else {
+  bool select(false);
+  if (trigbit_.size() > 0) {
```

Comment:
this collection is in RAW only, IIRC. Check if it is needed, as you request Trigger bits in the cfi.